sycl/source/detail/event_impl.hpp

```diff
@@ -351,6 +355,9 @@ class event_impl {
   // stored here.
   sycl::detail::pi::PiExtSyncPoint MSyncPoint;
 
+  // Signifies whether this event is
```

Comment:
comment reads as half finished

Reply:
You're absolutely right! It has been updated.